app/views/flood_risk_engine/enrollments/steps/_check_location.html.erb

```diff
@@ -1 +1,31 @@
-check_location
+<fieldset>
+  <legend class="visuallyhidden"><%= t('.legend') %></legend>
+
+  <div class="notice">
+    <i class="icon icon-important"><span class="visuallyhidden"><%= t('.text') %></span></i>
+    <strong class="bold-small"><%= t('.panel') %></strong>
+  </div>
+
+  <br>
+  <p class="panel panel-border-wide">
+    <%= t('.para_text') %>
+    <br>
+    <%= link_to 'Flood risk activities guidance', "TBD" %>
+  </p>
+
+
+  <%= form_group_and_validation(form, :base) do %>
+    <div class="form-group">
+      <label class="block-label" for="location_check_yes">
+        <%= f.radio_button :location_check, :yes, { checked: false, :class => "radio", id: "yes_option" } %>
+        <%= t('.label') %>
+      </label>
+
+      <label class="block-label" for="location_check_no">
+        <%= f.radio_button :location_check, :no, { checked: false, :class => "radio", id: "no_option" } %>
+        <%= t '.label1' %>
+      </label>
+  <% end %>
+  </div>
+
+</fieldset>
```

Comment:
Ok my thinking was, Once it gets to the StepController the initial step has been saved in the DB as check_location, indicating that Yes the Location has been checked and the journey under way.   I thought the Step saved in the DB was a step that has already been completed, this way it seems more clear the jounrey has started rather than creating an Enrollment with a nil state

It all works fine in the front end